PowerShell/JumpCloud Module/Public/Users/Remove-JCUser.ps1

```diff
@@ -67,52 +92,106 @@ UserID has an Alias of _id. This means you can leverage the PowerShell pipeline
             } else {
                 Throw "Username does not exist. Run 'Get-JCUser | Select-Object username' to see a list of all your JumpCloud users."
             }
+        } elseif ($PSCmdlet.ParameterSetName -eq 'UserID') {
+            # Validate if the Id is a JC User from the $UserHash
+            if ($UserHash.ContainsKey($UserID)) {
+                Write-Debug "UserID is a valid JumpCloud User"
+                $UserId = $UserHash.GetEnumerator().Where({ $_.Name -contains ($UserID) }).Name
+                $Username = $UserHash.GetEnumerator().Where({ $_.Name -contains ($UserID) }) | Select-Object -ExpandProperty Value | Select-Object -ExpandProperty username
+            } else {
+                Write-Error "User does not exist. Please enter a valid UserID."
```

Comment:
$UserId is not needed in this call since we are already validating with the if statement. 
![image](https://github.com/user-attachments/assets/101f9620-2c44-480e-a9c1-8a2add570ef8)

Reply:
In this case I'm asking your error messaging to be more specific — what user does not exist? The username of the user being deleted or the manager? 

Just noting that when we error we should be providing helpful messaging for customers, I'd argue we need to say what user (and specify the entered text ID or otherwise) does not exist. If I got this message while using the function, I'm not sure I'd know what user does not exist. Make sense?